util/feed.go

```diff
@@ -91,6 +91,9 @@ type Feed struct {
 // all Subscribers to the feed. Events published to a closed feed, or to a feed
 // with no Subscribers, will be ignored.
 func (f *Feed) Publish(event interface{}) {
+	if f == nil {
```

Comment:
ok. You have a bunch of those checks in `{server/status,storage}/feed.go`, those should be safe to remove when this lands.